components/project/hero/price-per-nft.js

```diff
@@ -0,0 +1,52 @@
+const _priceTiers = [
+    {
+        id: "bronze",
+        label: "Bronze",
+        price: 120000000
+    },
+    {
+        id: "silver",
+        label: "Silver",
+        price: 190000000
+    },
+    {
+        id: "gold",
+        label: "Gold",
+        price: 250000000
+    },
+]
+
+export default function PricePerNFT({ nftPriceE8S, priceTiers = [] }) {
+
+    if (_priceTiers.length > 1) {
```

Comment:
if a project has no tiers (past projects), it will display the standard price using the `nftPriceE8S` property. That gives us full backwards compatibility on the UI